e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -21,85 +21,106 @@ describe('team management', () => {
   const describeIAMV2 = iamVersion.match(/v2/) ? describe : describe.skip;
   const describeProjectsEnabled = iamVersion === 'v2.1' ? describe : describe.skip;
 
-  beforeEach(() => {
-    cy.adminLogin('/settings/teams').then(() => {
-      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
-      adminToken = admin.id_token;
-      cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
-      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
-    });
-    cy.restoreStorage();
-  });
-
   afterEach(() => {
     cy.saveStorage();
     cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
   });
 
-  it('lists system teams', () => {
-    cy.contains('Create Team');
-    cy.get('chef-sidebar')
-      .invoke('attr', 'major-version')
-      .then((obj: Cypress.ObjectLike) => {
-        switch (<string><Object>obj) {
-          case 'v2': {
-            cy.get('#table-container chef-th').contains('ID');
-            cy.get('#table-container chef-th').contains('Name');
-
-            const systemTeams = ['admins', 'editors', 'viewers'];
-            systemTeams.forEach(name => {
-            cy.get('#table-container chef-tr').contains(name)
-              .parent().parent().find('chef-control-menu').as('control-menu');
-            });
-            break;
-          }
-          default: {
-            cy.get('#table-container chef-th').contains('Name');
-            cy.get('#table-container chef-th').contains('Description');
-
-            const systemTeams = ['admins'];
-            systemTeams.forEach(name => {
-            cy.get('#table-container chef-tr').contains(name)
-              .parent().parent().find('chef-control-menu').as('control-menu');
-            });
+  context('no custom initial page state', () => {
+    beforeEach(() => {
+      cy.adminLogin('/settings/teams').then(() => {
+        const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+        adminToken = admin.id_token;
+        cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
+        cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+      });
+      cy.restoreStorage();
+    });
+
+    it('lists system teams', () => {
+      cy.contains('Create Team');
+      cy.get('chef-sidebar')
+        .invoke('attr', 'major-version')
+        .then((obj: Cypress.ObjectLike) => {
+          switch (<string><Object>obj) {
+            case 'v2': {
+              cy.get('#table-container chef-th').contains('ID');
+              cy.get('#table-container chef-th').contains('Name');
+
+              const systemTeams = ['admins', 'editors', 'viewers'];
+              systemTeams.forEach(name => {
+              cy.get('#table-container chef-tr').contains(name)
```

Comment:
If we do this we shouldn't need the `.parent().parent()`:
```typescript
    cy.get('#table-container').contains('chef-tr', name)
      .find('chef-control-menu')
```
[see here for another one of these](https://github.com/chef/automate/blob/322969e75683aa85f0850ddac5b97121873bdab9/e2e/cypress/integration/ui/settings/03_team_add_users.spec.ts#L112-L113)

...but I'm not sure what the `as('control-menu')` does here 🤔

Reply:
Heh agreed. Didn't write that one. That's an old test. I'll update it in follow up PR.

Reply:
Yup, with the construction I've proposed, we shouldn't have to assume how many calls to `parents()` we need.